ChangeLog

```diff
@@ -1,4 +1,26 @@
-* Tue Apr 28 2020 Klaus Wenninger <klaus.wenninger@aon.at> Pacemaker-2.0.4
+* Wed May 13 2020 Klaus Wenninger <klaus.wenninger@aon.at> Pacemaker-2.0.4-rc2
+- Changesets: 28
+- Diff: 28 files changed, 437 insertions(+), 130 deletions(-)
+
+- Changes since Pacemaker-2.0.4-rc1
+  + controller: properly detect remote node info requests
+  + fencer: avoid infinite loop if device is removed during operation
+  + fencer: avoid possible use-of-NULL when parsing metadata
+  + libfencing, fencer: Improve parameter handling for RHCS-style fence-agents
+                - add `port` or `plug` parameter according to metadata
+		- `plug` parameter shown as non-required in the metadata 
+  + libfencing: respect `pcmk_host_argument=none` on `validate`
```

Comment:
We can leave this one out since it doesn't actually take effect unless we make another change